### DISCUSSION TITLE
.ini file from example not recognized as being formatted properly

### DISCUSSION THREAD

**Discussion by DeanRider** [BrainEnergyLab/Inflammation-Index#24] [Fiji Plugin Issues]
I have a similar problem as others with the .ini file downloaded with the example image directory not being recognized as having the proper format. I'm not sure how it was resolved with the other user, but here is a screenshot of my attempt, the log file, and the .ini file
<img width="907" alt="Screen Shot 2023-01-24 at 8 21 51 PM" src="https://user-images.githubusercontent.com/94093691/214459483-4c41d2fb-cf19-4e54-87a3-860b55a32076.png">

Any suggestions? 
MacOS Monterey 12.0.1, in case that matters.

x.pixel.sz = 0.580000
y.pixel.sz = 0.580000
z.spacing = 1.000000000000
no.of.planes = 20.000000000000
frames.per.plane = 6.000000000000

**Comment by DKClarke** [comment]
Hi @DeanRider, I believe is actually the direct result of making a fix for the user with the other issue. This plugin is written in the ImageJ macro language and seems to do some weird things when parsing characters that depends on the environment its been run in. I've just uploaded a new plugin version (1.9) that should resolve your issue.

**Reply by DeanRider** [reply]
Thank you!
